src/screens/details/Moves.js

```diff
@@ -22,4 +49,8 @@ const styles = StyleSheet.create({
     height: '100%',
     alignItems: 'center',
   },
+  contentArea: {
+    width: '100%',
+    paddingHorizontal: 8,
```

Comment:
Instead of using this padding, for example lower width of components and add marginLeft. You'll avoid sitations like this one:

![image](https://user-images.githubusercontent.com/43967269/119472057-2cb1a000-bd4a-11eb-9a32-9d77560fe63a.png)

As you can see, the edges are not displayed fully. To compare, in the component below both left and right edge are well-displayed:

![image](https://user-images.githubusercontent.com/43967269/119472197-523ea980-bd4a-11eb-854a-9f1d9cb091ed.png)

What is more, make sure that the margins are the same in all of our components.

Reply:
I would make bigger margin between GameVersion and its value

Reply:
It's a bug. I came up with different work around.
https://github.com/facebook/react-native/issues/25703